manual/program/proposal_guidelines.rst

```diff
@@ -0,0 +1,56 @@
+Proposal Submission Guidelines
+==============================
+
+.. image:: tiptap.gif
+  :width: 400
+  :alt: Meme cat writing a SciPy proposal
+
+Here are the instructions and tips for submitting a proposal to SciPy.
+
+Instructions
+------------
+
+.. image:: tiptappy.gif
+  :width: 400
+  :alt: Another meme cat writing a SciPy proposal
+
+**Abstract**
+
+Your Abstract will appear in the online schedule and give attendees a sense of your talk. This should be around 100 words or less.
```

Comment:
I think in the past this was also the only part of the description that was in the printed program (though maybe we won't have a printed program this year?)

Reply:
Good point, let me check with Jim

Reply:
Jim says we'll just have an online-only program (web and PDF) so we can make last-minute changes more easily.